applications/trunkstore/src/ts_from_offnet.erl

```diff
@@ -306,6 +317,7 @@ routing_data(ToDID, AcctID, Settings) ->
                          ,{<<"To-IP">>, build_ip(ToIP, ToPort)}
                          ,{<<"Route-Options">>, RouteOpts}
                          ,{<<"Hunt-Account-ID">>, HuntAccountId}
+                         ,{<<"Authorizing-ID">>, wh_doc:id(Settings)} % connectivity doc id
```

Comment:
Is added to endpoint's CCVs [here](https://github.com/2600hz/kazoo/pull/1553/files#diff-4b15a7d704417b061680d835865cc059R205)